### PR TITLE
FLIP-1962: Update Central Hub API

### PR DESCRIPTION
**Pull request by mbaltrimas**

### CODE REVIEW

File: services/fl-service/src/services/fl-scheduler-service.ts [hunk 1]
```diff
@@ -377,6 +377,8 @@ export const prepareAndStartTraining = async (
             throw new Error("Failed to get the net endpoint");
         }
 
+        await validateClientAvailability(clients, endpoint, requestId);
+
         logger.object({
```
Comment (handyandypandyandy) [inline services/fl-service/src/services/fl-scheduler-service.ts:382]: can you move this log up to after where `bundleApplication()` is called?

File: services/fl-service/src/services/fl-service.ts [hunk 2]
```diff
@@ -407,25 +400,7 @@ export const startTraining = async (
          */
```
Comment (handyandypandyandy) [inline services/fl-service/src/services/fl-service.ts:400]: Can you remove the comments here or either update them to describe what the upload and submit job steps are?

File: services/fl-service/src/services/fl-scheduler-service.ts (hunk 1 above)
Comment (mbaltrimas) [reply services/fl-service/src/services/fl-scheduler-service.ts]: moved it up 👍

File: services/fl-service/src/services/fl-service.ts (hunk 2 above)
Comment (mbaltrimas) [reply services/fl-service/src/services/fl-service.ts]: added

**Review by handyandypandyandy — APPROVED** [review]

**Review by MichaelOSullivanAnswer — APPROVED** [review]

**Pull request merged**